Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6187.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Data;
+using System.Text;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 6187, "Visual Material Entry Underline color should reference PlaceholderColor",
+		PlatformAffected.All)]
+#if UITEST
+	[NUnit.Framework.Category(UITestCategories.Entry)]
+#endif
+	public class Issue6187 : TestContentPage
+	{
+		protected override void Init()
+		{
+			Visual = VisualMarker.Material;
+
+			Entry entry = new Entry
+			{
+				Placeholder = "Enter text...",
+				PlaceholderColor = Color.Green,
+				TextColor = Color.Red
+			};
+
+			StackLayout layout = new StackLayout
+			{
+				Padding = new Thickness(0, 50, 0, 0)
+			};
+
+			layout.Children.Add(entry);
+
+			Content = layout;
+		}
+
+
+
+#if UITEST
+		[Test]
+		public void ImageShouldLayoutOnTopOfButton()
+		{
+			
```

Comment:
No test?

Reply:
Yes, sorry, I did note that in the description: "No automated UI test as I do not know how to check the underline color of an Entry using UITest."

Reply:
We can probably just remove this code, then. Instead, we should add a label to the screen that tells the reviewer what to look for!

Reply:
Maybe also be a good candidate for the new tests suite that @hartez  is working on.

Reply:
I added a label describing expected behavior and removed the UITest code. 

I have not heard about any new test suite, but would be happy to learn and try it for this PR.